genesyscloud/tfexporter/tf_exporter_resource_test.go

```diff
@@ -109,7 +109,6 @@ func (r *registerTestInstance) registerTestResources() {
 	providerResources["genesyscloud_telephony_providers_edges_trunkbasesettings"] = gcloud.ResourceTrunkBaseSettings()
 	providerResources["genesyscloud_telephony_providers_edges_trunk"] = gcloud.ResourceTrunk()
 	providerResources["genesyscloud_user_roles"] = gcloud.ResourceUserRoles()
-	providerResources["genesyscloud_webdeployments_configuration"] = gcloud.ResourceWebDeploymentConfiguration()
```

Comment:
Shouldn't this be changed to point to the new package instead of being removed?

Reply:
Done.